docs/neo_fs_starter.md

```diff
@@ -0,0 +1,122 @@
+# NeoFS Quick Start Guide
```

Comment:
There are a lot of opinions on what a QSG is. We mostly think of it in terms of network deployment/storage node setup. This one is about using CLI for testnet interactions, so it should be `docs/testnet-starter.md` maybe.